components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml

```diff
@@ -121,12 +121,12 @@
         android:layout_height="wrap_content"
         android:layout_alignParentEnd="true"
         android:layout_marginTop="16dp"
-        android:backgroundTint="@color/button_state_list"
         android:text="@string/mozac_feature_prompt_save_confirmation"
         android:textAllCaps="false"
         android:textColor="?android:windowBackground"
         android:textSize="14sp"
         android:textStyle="bold"
+        app:backgroundTint="@color/button_state_list"
```

Comment:
Doesn't this require an `AppCompatButton`?

Reply:
It seems to work in my testing. My understanding of the support library is that it does some auto-swapping at layout time Button for -> AppCompatButton etc, but I can be explicit in the declaration